src/com/sap/piper/Utils.groovy

```diff
@@ -7,19 +7,31 @@ import groovy.text.GStringTemplateEngine
 import java.nio.charset.StandardCharsets
 import java.security.MessageDigest
 
+def stashAndLog(Map params) {
```

Comment:
Rename to `stash`.
For some reason my tests failed with strange behaviour. I'll try

Reply:
https://github.com/SAP/jenkins-library/pull/4380/commits/1667b65971c3dafef4e299a95cc86bfbb1c906e3